app/services/moves/finder.rb

```diff
@@ -38,8 +38,9 @@ def apply_filter(scope, param_name)
     def apply_date_range_filters(scope)
       scope = scope.where('date >= ?', filter_params[:date_from]) if filter_params.key?(:date_from)
       scope = scope.where('date <= ?', filter_params[:date_to]) if filter_params.key?(:date_to)
+      # created_at is a date/time, so inclusive filtering has to be subtlely different
       scope = scope.where('created_at >= ?', filter_params[:created_at_from]) if filter_params.key?(:created_at_from)
-      scope = scope.where('created_at <= ?', filter_params[:created_at_to]) if filter_params.key?(:created_at_to)
+      scope = scope.where('created_at <= ?', Date.parse(filter_params[:created_at_to]) + 1) if filter_params.key?(:created_at_to)
```

Comment:
should this be `'created_at < ?'` not `'created_at <= ?'` ?

Reply:
nope

Reply:
Hi @starswan this test fails because `<=` will include 00:00:00 in the following day: 

```
RSpec.describe Move do

  let!(:move1) { create(:move, created_at: '2000/01/01 00:00:00') }
  let!(:move2) { create(:move, created_at: '2000/01/02 00:00:00') }
  let!(:move3) { create(:move, created_at: '2000/01/03 00:00:00') }
  let!(:move4) { create(:move, created_at: '2000/01/04 00:00:00') }

  let(:created_at_from) {'2000/01/02'}
  let(:created_at_to) {'2000/01/03'}

  subject { Move
                .where('created_at >= ?', created_at_from)
                .where('created_at <= ?', Date.parse(created_at_to) + 1) }


  it { is_expected.to match_array([move2, move3]) }
end
```

Would you mind adding a test like this and fixing the scope ?